src/current/v25.1/set-up-logical-data-replication.md

```diff
@@ -85,87 +96,148 @@ When you run LDR in [`immediate` mode](#modes), you cannot replicate a table wit
+You can include multiple tables in the LDR stream for unidirectional or bidirectional setups. Ensure that the table name in the source table list and destination table list are in the same order:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICALLY REPLICATED TABLE ({database.public.destination_table_name_1}, {database.public.destination_table_name_2}) FROM TABLE ({database.public.source_table_name_1}, {database.public.source_table_name_2}) ON 'external://source' WITH bidirectional ON 'external://destination', label=track_job;
+~~~
+
+With the LDR streams created, move to [Step 4](#step-4-manage-and-monitor-the-ldr-jobs) to manage and monitor the jobs.
+
+#### `CREATE LOGICAL REPLICATION STREAM`
+
+Ensure you've created the table on the destination cluster with a matching schema definition to the source cluster table. From the **destination** cluster, start LDR. Use the fully qualified table name for the source and destination tables:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.source_table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.destination_table_name};
+~~~
+
+You can change the default `mode` using the `WITH mode = validated` syntax.
+
+If you would like to add multiple tables to the LDR job, ensure that the table name in the source table list and destination table list are in the same order:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLES ({database.public.source_table_name_1},{database.public.source_table_name_2},...)  ON 'external://{source_external_connection}' INTO TABLES ({database.public.destination_table_name_1},{database.public.destination_table_name_2},...);
+~~~
+
+(**Optional**) At this point, you've set up one LDR stream from cluster A as the source to cluster B as the destination. To set up LDR streaming in the opposite direction using `CREATE LOGICAL REPLICATION STREAM`, run the statement again but cluster B will now be the source, and cluster A will be the destination.
+
+## Step 4. Manage and monitor the LDR jobs
 
-1. Move on to [Step 4](#step-4-optional-set-up-bidirectional-ldr) to set up a second LDR job. Or, once you have set up your required LDR jobs, refer to [Step 5](#step-5-monitor-the-ldr-jobs) to monitor the jobs in the DB Console.
+Once LDR has started, an LDR job will run on the destination cluster. You can [pause]({% link {{ page.version.version }}/pause-job.md %}), [resume]({% link {{ page.version.version }}/resume-job.md %}), or [cancel]({% link {{ page.version.version }}/cancel-job.md %}) the LDR job with the job ID. Use [`SHOW LOGICAL REPLICATION JOBS`]({% link {{ page.version.version }}/show-logical-replication-jobs.md %}) to display the LDR job IDs:
 
-## Step 4. (Optional) Set up bidirectional LDR
+{% include_cached copy-clipboard.html %}
```

Comment:
I just realized I did something bad: I should have added a release note to this SHOW LOGICAL REPLICATION JOBS [change](https://github.com/cockroachdb/cockroach/pull/139030) I made for 25.1, which means we'll need to update the examples in the docs. Should I open a separate docs issue?

This is another SHOW LDR JOBS [change](https://github.com/cockroachdb/cockroach/pull/134339) that should have had a release note as well.

Reply:
No worries! I've updated both of these column name changes and added the release notes to the 25.1 testing releases they released with.

Reply:
thank you!